.github/workflows/automerge.yml

```diff
@@ -0,0 +1,27 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: 'pascalgn/automerge-action@v0.11.0'
```

Comment:
I don't understand how does this work? I mean how does it know that this PR is from dependabot? I don't see the label check here

Reply:
The default label it looks for is automerge

Reply:
Nice! Thanks :)